modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -57,11 +57,11 @@ To install on {ibm-z-name} under {op-system-base} KVM, you need:
 *   A {op-system-base} KVM host configured with an OSA or RoCE network adapter.
 *   Either a {op-system-base} KVM host that is configured to use bridged networking in libvirt or MacVTap to connect the network to the guests.
 +
-See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/configuring_and_managing_virtualization/index#types-of-virtual-machine-network-connections_configuring-virtual-machine-network-connections[Types of virtual network connections].
+See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/configuring_and_managing_virtualization/index#virtual-networking-network-address-translation_types-of-virtual-machine-network-connections[Types of virtual network connections].
```

Comment:
```suggestion
See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/configuring_and_managing_virtualization/index#virtual-networking-network-address-translation_types-of-virtual-machine-network-connections[Types of virtual network machine connections].
```

Reply:
The link seems to float on its own but is it relevant to both bullet points or just one? If one, maybe it should be moved to that line.